security/webauthn/src/test/java/io/quarkus/ts/security/webauthn/AbstractWebAuthnTest.java

```diff
@@ -23,10 +28,13 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public abstract class AbstractWebAuthnTest {
 
+    private URL url;
+
     protected abstract RestService getApp();
 
+    private static final String REGISTER_CHALLENGE_OPTIONS_URL = "/q/webauthn/register-options-challenge";
     private static final String REGISTER_URL = "/q/webauthn/register";
-    private static final String REGISTER_CALLBACK_URL = "/q/webauthn/callback";
+    private static final String LOGIN_CHALLENGE_OPTIONS_URL = "/q/webauthn/login-options-challenge";
```

Comment:
This doesn't seem to be used.

Did you have any plans with it or you wanted to have it here for completeness?

Reply:
For completeness, but you're right it's not used. Neither was the `/q/webauthn/login` endpoint defined there.